src/libraries/Microsoft.Extensions.Configuration.Binder/tests/Common/ConfigurationBinderTests.cs

```diff
@@ -2404,5 +2405,27 @@ public void SharedChildInstance()
             config.GetSection("A").Bind(instance);
             Assert.Equal("localhost", instance.ConnectionString);
         }
+
+        [Fact]
+        public void CanBindToMockConfiugrationSection()
```

Comment:
Looks need to exclude the test for wasm?

Reply:
Yep will do this: https://github.com/dotnet/runtime/blob/736dabeca728ccf8b911d96d1b4c575b4d0db7d2/src/libraries/Microsoft.Extensions.Primitives/tests/CompositeChangeTokenTest.cs#L45-L46

Reply:
So that didn't seem to be enough to fix it since the failure appears to be happening when the tool chain runs on the test.  I'll look closer at other tests.  I really don't want to disable the entire assembly, nor do I want to implement an verbose hand-written mock configuration for this.

Reply:
I tried a hand-written mock and was able to get it to work.  I'll use this method since adding the Moq dependency makes us lose test coverage.

Reply:
Sounds good.